app/uk/gov/hmrc/cdsreimbursementclaim/models/claim/CompleteClaim.scala

```diff
@@ -51,330 +51,80 @@ object CompleteClaim {
 
-    def reasonAndBasisOfClaim: Option[CompleteReasonAndBasisOfClaimAnswer] = completeClaim match {
-      case CompleteC285Claim(
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            maybeCompleteReasonAndBasisOfClaimAnswer,
-            _,
-            _,
-            _,
-            _,
-            _
-          ) =>
-        maybeCompleteReasonAndBasisOfClaimAnswer
-    }
+    def reasonAndBasisOfClaim: Option[CompleteReasonAndBasisOfClaimAnswer] =
+      completeClaim.get(_.maybeCompleteReasonAndBasisOfClaimAnswer)
 
-    def bankDetails: Option[CompleteBankAccountDetailAnswer] = completeClaim match {
-      case CompleteC285Claim(
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            maybeCompleteBankAccountDetailAnswer,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _
-          ) =>
-        maybeCompleteBankAccountDetailAnswer
-    }
+    def bankDetails: Option[CompleteBankAccountDetailAnswer] =
+      completeClaim.get(_.maybeCompleteBankAccountDetailAnswer)
 
-    def entryDeclarationDetails: Option[CompleteDeclarationDetailsAnswer] = completeClaim match {
-      case CompleteC285Claim(
-            _,
-            _,
-            _,
-            maybeCompleteDeclarationDetailsAnswer,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _
-          ) =>
-        maybeCompleteDeclarationDetailsAnswer
-    }
+    def entryDeclarationDetails: Option[CompleteDeclarationDetailsAnswer] =
+      completeClaim.get(_.maybeCompleteDeclarationDetailsAnswer)
 
-    def duplicateEntryDeclarationDetails: Option[CompleteDuplicateDeclarationDetailsAnswer] = completeClaim match {
-      case CompleteC285Claim(
-            _,
-            _,
-            _,
-            _,
-            maybeCompleteDuplicateDeclarationDetailsAnswer,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _,
-            _
-          ) =>
-        maybeCompleteDuplicateDeclarationDetailsAnswer
-    }
+    def duplicateEntryDeclarationDetails: Option[CompleteDuplicateDeclarationDetailsAnswer] =
+      completeClaim.get(_.maybeCompleteDuplicateDeclarationDetailsAnswer)
 
-    def evidences: SupportingEvidenceAnswer = completeClaim match {
-      case CompleteC285Claim(_, _, _, _, _, _, _, _, _, _, supportingEvidenceAnswer, _, _, _, _, _, _, _) =>
-        supportingEvidenceAnswer
-    }
+    def evidences: SupportingEvidencesAnswer =
```

Comment:
I thought we discussed that you were going to combine the supporting-evidences and schedule-mrn document at this point or is that going to be done in another ticket?

Reply:
Yes, I thought we agreed that this will be done in the other ticket, specially once review page will be created. I thought it was understandable that currently all I can do is initiate schedule then redirect to supporting evidence pages.